che-mount/entrypoint.sh

```diff
@@ -93,45 +113,105 @@ trap 'stop_sync' SIGHUP SIGTERM SIGINT
 init_logging
 init_global_variables
 parse_command_line "$@"
-status=$?
-if [ $status -ne 0 ]; then
-    exit 1
+[ "$(ls -A /mnthost )" ] && EMPTY=false || EMPTY=true
+if [ $EMPTY = false ]; then
+    warn "(che mount): Local folder is not empty. This could overwrite folders/files on local desktop machine or remote workspace machine. Are you sure you want to continue?"
```

Comment:
Any thoughts on the new message. I am worried if I don't example the risks that a user could lose data. I could split the message into multiple lines with "\n".